hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/OzoneQuota.java

```diff
@@ -137,9 +134,10 @@ public static OzoneQuota parseQuota(String quotaString)
     }
 
     if (!found) {
-      throw new IllegalArgumentException(
-          "Quota unit not recognized. Supported values are BYTES, MB, GB and " +
-              "TB.");
+      String exceptionInfo =
+          "Quota unit not recognized. " +
+          "Supported values are BYTES, MB, GB and TB.";
+      throw new IllegalArgumentException(exceptionInfo);
```

Comment:
Let us avoid creating the variable `exceptionInfo` as it is not used after the next line.
We could still beautify the current line as:
``` suggestion
      throw new IllegalArgumentException("Quota unit not recognized." +
             "Supported values are BYTES, MB, GB and TB.");
```

Reply:
Thanks @dineshchitlangia for the advice.
Fixed.